onnx/cpp2py_export.cc

```diff
@@ -577,31 +577,39 @@ PYBIND11_MODULE(onnx_cpp2py_export, onnx_cpp2py_export) {
   auto shape_inference = onnx_cpp2py_export.def_submodule("shape_inference");
   shape_inference.doc() = "Shape Inference submodule";
   py::register_exception<InferenceError>(shape_inference, "InferenceError");
+  py::register_exception<TypeInferenceError>(shape_inference, "TypeInferenceError");
```

Comment:
Just making sure: Are we subclassing InferenceError?

Reply:
yes we are subclassing TypeInferenceError and ShapeInferenceError from InferenceError.